apps/explorer/src/app/routes/oracles/index.tsx

```diff
@@ -33,15 +34,28 @@ const ORACLE_SPECS_QUERY = gql`
 `;
 
 const Oracles = () => {
-  const { data } = useQuery<OracleSpecsQuery>(ORACLE_SPECS_QUERY);
+  const { hash } = useLocation();
+  const { data, loading } = useQuery<OracleSpecsQuery>(ORACLE_SPECS_QUERY);
+
+  useEffect(() => {
```

Comment:
question (blocking): Annoying that we need this effect. Does linking with a hash not work when using react-router-dom?

Reply:
it does, but only when the data is already on the page. when we have a loading state, the router doesn't automatically recheck the contents, so i have to tell it to do so